packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -2,11 +2,10 @@ import React, { useEffect, useState, useCallback } from 'react';
 import {
   generateUUID,
   InvoiceLineNodeType,
-  Item,
```

Comment:
Outbounds in general: Also just changed from the use of `Item` in the edit modal to use `ItemRowFragment`